backend/src/private-message/private-message.resolver.spec.ts

```diff
@@ -1,19 +1,515 @@
+      }
+      const resExpected = { id: '1', ...data }
+      privateMessageService.create.mockReturnValue(resExpected)
+
+      const result = await privateMessageresolver.createPrivateMessage(data)
+
+      expect(result).toStrictEqual(resExpected)
+    })
+    it('updatePrivateMessage', async () => {
+      const data: UpdatePrivateMessageInput = {
+        content: 'this is a message'
+      }
+      const resExpected = { id: '1', ...data }
+      privateMessageService.update.mockReturnValue(resExpected)
+
+      const result = await privateMessageresolver.updatePrivateMessage(
+        '1',
+        data
+      )
+      expect(privateMessageService.update).toHaveBeenCalledWith('1', data)
+    })
+    it('deletePrivateMessage', async () => {
+      const resExpected = { id: '1' }
+      privateMessageService.delete.mockReturnValue(resExpected)
 
-    resolver = module.get<PrivateMessageResolver>(PrivateMessageResolver)
+      const result = await privateMessageresolver.deletePrivateMessage('1')
+      expect(result).toStrictEqual(resExpected)
+    })
   })
+  describe('Test Query', () => {
+    it('findOnePrivateMessage', async () => {
+      const resExpected = {
+        senderId: '1',
+        receiverId: '2',
+        content: 'this is a message1'
+      }
+      privateMessageService.findOne.mockReturnValue(resExpected)
+      const result = await privateMessageresolver.findOnePrivateMessage('1')
+
+      expect(result).toStrictEqual(resExpected)
+      expect(privateMessageService.findOne).toHaveBeenCalledWith('1')
+    })
+    it('findAllPrivateMessageWith', async () => {
+      const resExpected = {
+        senderId: '1',
+        receiverId: '2',
+        content: 'this is a message'
+      }
+      privateMessageService.findAllMessageWith.mockReturnValue(resExpected)
+
+      const result = await privateMessageresolver.findAllPrivateMessageWith(
+        '1',
+        '2'
+      )
+
+      expect(result).toStrictEqual(resExpected)
+      expect(privateMessageService.findAllMessageWith).toHaveBeenCalledWith(
+        '1',
+        '2'
+      )
+    })
+    it('findAllMessageWithLiteVersion', async () => {
+      const resExpected = {
+        senderId: '1',
+        receiverId: '2',
+        content: 'this is a message'
+      }
+
+      privateMessageService.findAllMessageWithLiteVersion.mockReturnValue(
+        resExpected
+      )
+
+      const result =
+        await privateMessageresolver.findAllPrivateMessageWithLiteVersion(
+          '1',
+          '2'
+        )
+
+      expect(result).toStrictEqual(resExpected)
+      expect(
+        privateMessageService.findAllMessageWithLiteVersion
+      ).toHaveBeenCalledWith('1', '2')
+    })
+    it('findPrivateMessageContain', async () => {
+      const resExpected = {
+        senderId: '1',
+        receiverId: '2',
+        content: 'this is a message needle'
+      }
+
+      privateMessageService.findPrivateMessageContain.mockReturnValue(
+        resExpected
+      )
+
```

Comment:
Here we made a mistake, replace in resolver file too, we want multiple message, so it's findAllPrivateMessageContain, not just find.
ResExcpeted should be an array too